cmd/catalyst-node/catalyst-node.go

```diff
@@ -434,25 +444,47 @@ func redirectHlsHandler() http.Handler {
 		lat := r.Header.Get("X-Latitude")
 		lon := r.Header.Get("X-Longitude")
 
-		nodeAddr, err := getClosestNode(playbackID, lat, lon)
-		if err != nil {
+		var nodeAddr, validPrefix string
+		var err error
+
+		for _, prefix := range redirectPrefixes {
+			nodeAddr, err = getClosestNode(playbackID, lat, lon, prefix)
+			if err != nil {
+				glog.Errorf("error finding origin server playbackID=%s prefix=%s error=%s", playbackID, prefix, err)
```

Comment:
I think we should not log error if we got just one error for one prefix. IFAIU the situation in which Mist does not have `video+<playback_id>` but has `videorec+<playback_id` is completely normal and should not result in an error.

Reply:
makes sense. will update this.

Reply:
verbosity logs only support `Info` messages. updated this in next commit.